moderation/moderation.md

```diff
@@ -9,3 +9,28 @@ has_children: true
+| Kick          | `/kick [ID/Mention] (Reason)`               | Kicks the user, notifying them through DMs and logging the incident                                                                                                                                                                                                |
+| Modlogs       | `/modlogs [ID/Mention]`                     | Shows all moderation logs of the user.                                                                                                                                                                                                                             |
+| Modnote       | `/modnote [ID/Mention] [Reason]`            | Adds a mod note onto the user's logs; This note is only visible to moderators, and does not DM the user or show when they types /mylogs                                                                                                                            |
+| Modstats      | `/modstats [ID/Mention]`                    | Gets statistics of the specified moderator. You must be a moderation manager or admin to use this command unless you are invoking it on yourself                                                                                                                   |
+| Mute          | `/mute [ID/Mention] (Duration) (Reason)`    | Mutes the user, notifying them through DMs when the mute begins and ends and logging the incident. Not specifying a duration sets the mute to permanent                                                                                                            |
+| Presetreasons | `/presetreasons (preset)`                   | Lists all available preset reason variables for the server. If a preset variable is specified, this command will list the correlated preset reason                                                                                                                 |
+| Purge         | `/purge [Amount <= 1000]`                   | Mass-deletes the specified number of messages in the channel invoked. This command has a limit of 1000 messages, and may take a few seconds. You must be a moderation manager or admin to use this command                                                         |
+| Reason        | `/reason [Case Num] [New Reason]`           | Edits the reason of an existing moderation case. This will also edit the DM for the user who received the action to reflect the change. You may only edit your own cases unless you are a moderation manager                                                       |
+| Revoke        | `/revoke [Case Num] (Reason)`               | Revokes the specified moderation case. This will prevent the case from showing on /modlogs and will edit the DM for the user who received the action to reflect that the action is revoked. You may only revoke your own cases unless you are a moderation manager |
```

Comment:
```suggestion
| Revoke        | `/revoke [Case Num] (Reason)`               | Revokes the specified moderation case. This will prevent the case from showing on /modlogs and will edit the DM for the user who received the action to reflect that the action is revoked. You may only revoke your own cases unless you are a moderation manager or admin. |
```